pkg/controller/user/importbatch.go

```diff
@@ -105,3 +106,30 @@ func (c *Controller) HandleImportBatch() http.Handler {
 		c.h.RenderJSON(w, http.StatusOK, response)
 	})
 }
+
+func (c *Controller) sendInvitation(ctx context.Context, toEmail string) error {
+	realm := controller.RealmFromContext(ctx)
+
+	// Fallback to Firebase
```

Comment:
Nit: logically, it might make sense to put this at the end (and change the conditional to != nil, since it's the "fallback"